src/Commands/Modules/NonEssentialModules.php

```diff
@@ -0,0 +1,238 @@
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class NonEssentialModules extends Command
+{
+    /**
+     * @var array possible allowed command
+     */
+    private const ALLOWED_COMMAND = ['status', 'uninstall', 'install'];
+
+    private const NON_ESSENTIAL_MODULES = ['emarketing', 'gamification', 'ps_checkout', 'ps_eventbus', 'ps_metrics', 'psaddonsconnect', 'statsvisits', 'welcome'];
+
+    private const STATS_MODULES = ['statsbestcategories', 'statsbestcustomers', 'statsbestmanufacturers', 'statsbestproducts',
+        'statsbestsuppliers', 'statsbestvouchers', 'statscarrier', 'statscatalog', 'statscheckup', 'statsdata',
+        'statsequipment', 'statsforecast', 'statslive', 'statsnewsletter', 'statsorigin', 'statspersonalinfos',
+        'statsproduct', 'statsregistrations', 'statssales', 'statssearch', 'statsstock', 'statsvisits', ];
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('fop:modules')
+            ->setDescription('Manage non-essential and stats modules.')
+            ->setHelp('This command Uninstall or Install non-essential and stats modules.')
+            ->addArgument(
+                'action',
+                InputArgument::OPTIONAL,
+                '( possible values : ' . $this->getPossibleActions() . ') ',
+                'status'
+            )
+            ->addOption(
+                'modulename',
+                null,
+                InputOption::VALUE_OPTIONAL,
+                'Module to Install / Uninstall'
+            )
+            ->addOption(
+                'idsmodule',
+                null,
+                InputOption::VALUE_OPTIONAL,
+                'IDs module to Install or Uninstall separate by coma'
+            );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $io = new SymfonyStyle($input, $output);
+        $moduleManager = $this->getContainer()->get('prestashop.module.manager');
+        $action = $input->getArgument('action');
+        $modulesInfos = [];
+        $modulesStatsInfos = [];
+
+        switch ($action) {
+            case 'status':
+                $io->text('<info>Non-essential Modules Informations</info>');
+
+                foreach (self::NON_ESSENTIAL_MODULES as $key => $uselessModule) {
+                    $modulesInfos[] = ['id' => $key, 'name' => $uselessModule, 'present' => $this->moduleExist($uselessModule) ? 'yes' : 'no', 'installed' => $moduleManager->isInstalled($uselessModule) ? 'yes' : 'no'];
+                }
+
+                $io->table(['ID', 'Name', 'Present?', 'Installed?'], $modulesInfos);
+                $io->text('You can'
+                    . PHP_EOL . '    - uninstall all modules by running  : `./bin/console fop:modules uninstall`'
```

Comment:
fop:modules uninstall
I prefer something more explicit like
`fop:modules:essentials uninstall`

Reply:
ok

Reply:
Je me penche sur ça, du coup il faudrait que je fasse deux fichiers séparer les stats et les not-essential est ce que :  `fop:modules uninstall-not-essential` est bon pour toi.

Reply:
la convention est fop:domain:action
Ca serait donc `fop:modules:uninstall not-essential` et `fop:modules:uninstall stats`

Reply:
tu peux le faire dans le meme fichier